src/js/views/settingsView.js

```diff
@@ -0,0 +1,124 @@
+var Marionette = require('backbone.marionette'),
+    _ = require('underscore'),
+    $ = require('jquery'),
+    bindAll = require('lodash.bindall');
```

Comment:
I've seen @lemieux and @juliangarritano use this multi line var syntax. I don't like it, this guy explains why: (http://benalman.com/news/2012/05/multiple-var-statements-javascript/)

I've refrained from commenting (until now) because its not a huuuge deal, If it was just my opinion I was arguing I'd let it slide. However, the separate var style, eg:

```
var Marionette = require('backbone.marionette');
var _ = require('underscore');
```

is far more common across the SK codebase, and because style *consistency* across code bases is far more important than any one person's opinion about style I would suggest that supportkit-js should follow suit. You can use newlines to separate 3rd party from views from controllers etc.

I would further suggest that if possible we automate confirming code with tools like jsfmt @dannytranlx is there an option for this?